src/migrations/00102-device-add-secureboot-fde-status.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "device" ADD COLUMN IF NOT EXISTS "is secureboot enabled" BOOLEAN NULL;
+ALTER TABLE "device" ADD COLUMN IF NOT EXISTS "is storage encrypted" BOOLEAN NULL;
+ALTER TABLE "device" ADD COLUMN IF NOT EXISTS "secureboot keys metadata" JSONB NULL;
```

Comment:
This doesn't actually match the generated schema from the sbvr so this would be an invalid migration, it's better to fix at the sbvr level but it's worth noting that you can generate the schema via `npm run generate-model-schema` and make sure the column types match